### ISSUE TITLE
Feature request: ISO week date format

### ISSUE THREAD

**Issue by szabogyula** [kochrt/timeline-maker#150]
It should be useful to get ISO week date format support, like 2025-W12

https://en.wikipedia.org/wiki/ISO_week_date

**Comment by qalisto** [comment]
date format flexibility is essential for an app of this type to be taken seriously.

**Comment by itsdanott** [comment]
Agreed - having calendar week support would be very neat as some organization do sprint planning using iso weeks.

**Comment by kochrt** [comment]
Added in the parser [0.16.7](https://github.com/mark-when/parser/commit/c7341e1dba20ffaf51cc916556f0e0c4163aa8a0) and meridiem 0.2.18
